src/js/components/sortable.js

```
@@ -222,63 +255,192 @@ export default {
-            addClass(this.$el.children, this.clsItem);
+        getSortable(element) {
+            do {
+                const sortable = this.$getComponent(element, 'sortable');
+
+                if (
+                    sortable &&
+                    (sortable === this || (this.group !== false && sortable.group === this.group))
+                ) {
+                    return sortable;
+                }
+            } while ((element = parent(element)));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)